go.mod

```diff
@@ -67,4 +72,4 @@ require (
 	sigs.k8s.io/yaml v1.1.0 // indirect
 )
 
-replace github.com/miekg/dns v1.1.3 => github.com/miekg/dns v1.1.15
```

Comment:
We want to leave 1.15 alone, as it is used to rewrite caddy's miekg/dns dependency to match the one CoreDNS depending on.